### PR TITLE
blog post from rgsoc team rookies[1]

### PR DESCRIPTION
**Pull request by flyjwayur**
Hello :) 
We're HyeSoo & Tehetena from RGSoC 2016 team Rookies. 
We want to greet Hoodie community and share our story with the community in the Hoodie blog. 
We've been making 'Hoodie desktop application'.
It's been an amazing experience with Hoodie and we had so many helps from mentors.
If there are anything to edit, feel free to let us know. 
Thank you for the wonderful opportunity:D

### CODE REVIEW

File: _posts/2016-08-05-rgsoc-team-rookies-month-1.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Greeting from RGSOC 2016 team Rookies :D
+author: Tehetena & HyeSoo 
+layout: blog-post
+//permalink: /2013/06/Greeting-from-RGSOC-2016-team-Rookies/
+---
+Hi, we’re Tehetena and HyeSoo. We are happy to write about our 2016 summer experience with Hoodie project. We met Hoodie with Rails Girls summer of Code. We joined the Rails Girls summer of Code as a team ‘Rookies’. It has been an amazing experience, since we have started working on the Hoodie desktop application project.
```
Comment (varjmes) [inline _posts/2016-08-05-rgsoc-team-rookies-month-1.md]: "with the Hoodie project"

File: _posts/2016-08-05-rgsoc-team-rookies-month-1.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+When we started the project, we had low self confidence. Even though we have taken classes on different programming languages, but when it comes to working with it we always thought that we did not have all the knowledge it took to get the job done. Now we have understood that every working experience is a great opportunity to teach ourselves new things everyday.
+
+The very first lesson we learned was Git. We have done several school projects that we have managed to save and clone project repositories on Git, but after we have started working on the hoodie desktop application, there have been many challenges. At first, we did not know how to propose issues, pull request, fetch from remote, rebase and so on, and we were confused with the concept between local and remote. With our mentor Gregor’s help and many trials & errors, we now have better understanding of how the Github actually works and how it is used in our project and also when many contributors are working on the same project.
```
Comment (varjmes) [inline _posts/2016-08-05-rgsoc-team-rookies-month-1.md]: 'hoodie' should be capitalised :)

**Comment by varjmes** [comment]
This is fantastic! Congrats HyeSoo on the baby! ✨

File: _posts/2016-08-05-rgsoc-team-rookies-month-1.md (hunk 1 above)
Comment (flyjwayur) [inline _posts/2016-08-05-rgsoc-team-rookies-month-1.md]: Charlotteis, thank you so much:) I fixed it

File: _posts/2016-08-05-rgsoc-team-rookies-month-1.md (hunk 2 above)
Comment (flyjwayur) [inline _posts/2016-08-05-rgsoc-team-rookies-month-1.md]: Thank you so muchh :D I changed from hoodie to Hoodie :) and thank you for the congrats. Best wishes for you too.

**Comment by gr2m** [comment]
LGTM 👍 ping @renrutnnej

**Comment by jennwrites** [comment]
This is a great post! Thank you so much @flyjwayur and Tehetena!

I made edits so I'm giving this my LGTM so we can move forward with publishing.

I think it should go live around UTC+2 (is that right @janl?) on Wednesday, August 10. 

Thanks again Team Rookies ☺️

**Comment by machikoyasuda** [comment]
Hello @flyjwayur & @titay2 from Los Angeles 👋 

I'm a developer and former journalist in LA. I used to edit writing for a living. Now I write code and organize meetups for people like you <3 I started my coding journey in April of 2013 at the first Rails Girls in Los Angeles, so Rails Girls Summer of Code has a special place in my heart <3 💎  

In my years since then, I've noticed two things: (1) Code learning newbies learn so much from stories and blog posts like this - learning stories. (2) And thus, keeping a log like this is so important, not just for you two, but for other learners too. I'm definitely going to be sharing this post with our LA @fullstackla meetup group pals.

To make the post even more understandable for newbies, I have added some tiny little text edits line-by-line, if you so please! Let me know if you have any questions ever :)

File: _posts/2016-08-05-rgsoc-team-rookies-month-1.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+We started working on the Hoodie desktop application project. For making it, we’ve been working on UI/UX for the application and learning how to use Git, Node, localStorage and APIs to implement functionalities. 
+[Gregor](https://twitter.com/gr2m) is our project mentor. [Esko](https://twitter.com/orfjackal), [Omid](https://twitter.com/omidfi) and [Musse](https://github.com/mussebekabil) are our coaches, and [Pilar](https://twitter.com/Althaire) is our project supervisor. 
+
+![Kick-off meeting with mentor, coaches](/blog/images/201608/team-Rookies-kickoffcall.jpg)
+<font color="grey"><small><i>Rookies team in the main, at the bottom-right corner, from the left, are Esko, Gregor, Omid and Pilar. (Image: Pilar Huidobro)</i></small></font>
+
+## What we’ve achieved so far
+When we started the project, we had low self confidence. Even though we have taken classes on different programming languages, when it came to working with it we thought we did not have all the knowledge it took to get the job done. Now we understand that __every working experience is a great opportunity to teach ourselves new things everyday__.
```
Comment (machikoyasuda) [inline _posts/2016-08-05-rgsoc-team-rookies-month-1.md:21]: `self-confidence` is usually hyphenated, like this: http://www.dictionary.com/browse/self-confidence

File: _posts/2016-08-05-rgsoc-team-rookies-month-1.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Other than using GitHub, we have also learned a lot on __UI building__ and how to implement functionalities on the application using HTML, CSS, and JavaScript. We learned how to use EventListener, event.target, .map, .forEach, and localStorage. Using localStorage and making an API were very interesting. We followed the [tutorials](https://github.com/gr2m/localstorage-tutorial/) made by Gregor. It was very helpful and easy to follow. After practicing the code with tutorials, we had a call to solve the problems together and had a Q&A session. We have a call almost everyday. It’s been really helpful and given us deeper insight on how coding can be applied in the application and how we can use it in the desktop application with the Electron framework.
```
Comment (machikoyasuda) [inline _posts/2016-08-05-rgsoc-team-rookies-month-1.md:25]: Newbies might not know what UI stands for. You can add the abbreviation like `UI (user interface)`.

**Comment by machikoyasuda** [comment]
Ps. Congrats on the baby!!!!!!!!!!!!!!!!!!

**Comment by espy** [comment]
Merrrrged: http://hood.ie/blog/rgsoc-team-rookies-month-1.html 🎉

**Pull request merged**